phpstan.neon

```diff
@@ -1,11 +1,14 @@
 includes:
 	- vendor/phpstan/phpstan-doctrine/extension.neon
+	- vendor/proget-hq/phpstan-phpspec/extension.neon
```

Comment:
Is it added to the composer?

Reply:
You're right, wrong rebase....

Reply:
@lchrusciel Thank you, I didn't understand why Travis build was red.